packages/agent/src/agents/modes/utils.ts

```
@@ -0,0 +1,189 @@
+}
+
+export function isTokenLimitError(error: any): boolean {
+  return (
+    error.message?.includes('token limit') ||
+    error.message?.includes('tokens exceed') ||
+    error.message?.includes('context length')
+  );
+}
+
+// --- ERROR HANDLING --- //
+export function handleModelError(error: any): {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)